### ISSUE TITLE
Cannot get refresh-token 

### ISSUE THREAD

**Issue by zsocap** [Andre0512/lidl-plus#2]
After adding .items() inside the _register_link function in lidlplus/api.py here:

![kép](https://user-images.githubusercontent.com/108118952/210390144-a3baea61-2eba-45e6-82b4-c4f5106bcb34.png)

I was able to navigate to the page, but after providing the e-mail and password, then hitting the next button nothing happened:

![kép](https://user-images.githubusercontent.com/108118952/210390521-1d08955d-5069-4aea-b171-c5a159678130.png)

**Comment by Andre0512** [comment]
Hi, which country and language do you use? Than I try to reproduce it later, only tested german so far 😅

**Comment by zsocap** [comment]
Hi there, I used the following setup: 

lidl = LidlPlusApi(language="HU", country="hu")

It seems like the valid setup for Hungary.

**Comment by Andre0512** [comment]
Problem should be fixed now, I created a new release.
Sorry, v0.2.1 couldn't run, I had to test the releases more careful :facepalm:

**Comment by zsocap** [comment]
Thx very much for the update! 

May I ask you what should happen after you hit the submit button? I receive the following pop-up:

![image](https://user-images.githubusercontent.com/108118952/210529807-25f9adfe-0985-4547-a13d-96ec91715273.png)

Even if I put a break point to avoid the TimeoutException with the "Wrong credentials" message, and accept this pop-up manually, the submit button doesn't do anything. 

I am using ubuntu 20.04, with chrome. Tried it win 10, firefox, chrome as well, but it was the same.

**Comment by zsocap** [comment]
Ahhh, got it! So basically you get the token from the latest request's header data and I just have to skip this part: 

![image](https://user-images.githubusercontent.com/108118952/210555700-bf23d903-068b-4bf4-85dd-7ffa58981bbb.png)

I was able to get the token and use it to get my tickets. 

Checked my account on the phone, but I wasn't able to find anything related to 2fa. Maybe in Germany there are more options, e.g. online shopping and other functionalities that require more secure authentication. Thx for the help!!
